website/src/components/discourse/index.js

```diff
@@ -134,6 +134,14 @@ export const DiscourseFeed = ({
                   </span>
                 </>
               )}
+              {(topic?.blurb) && (
+                <>
+                  {' '}
+                  <p>
+                    {topic.blurb}
+                  </p>
```

Comment:
Paragraphs have a default margin-bottom throughout the site, so this creates an uneven spacing for each topic row.

I don't think it looks too bad as is, but if you wanted to adjust this, you can add the following css to `website/src/components/discourse/styles.module.css`:
```
.discourseTopics p {
  margin-bottom: 0;
}
```

![image](https://user-images.githubusercontent.com/13647110/193845783-1b66bbe3-43d1-4a97-8f4b-9f27c0c79526.png)